.github/workflows/test.yml

```diff
@@ -0,0 +1,66 @@
+name: Tests
+
+on: [push]
```

Comment:
You may also want to run on pull requests. See e.g. here: https://github.com/extendr/rextendr/blob/1ab743759698528a08fec7847df110ed9648249b/.github/workflows/R-CMD-check.yaml#L1-L9